README.md

```diff
@@ -129,67 +132,67 @@ public class DefaultController : Controller
+1. The single bundle per application - it's name can be set by `DefaultBundle` parameter (`default` by default):
 
   ```xml
   <bemSettings Mapper="Single" DefaultBundle="index" />
   ```
-1. Отдельный бандл для каждого серверного контроллера:
+1. Separate bundle for each server-side controller:
 
   ```xml
   <bemSettings Mapper="ByController" />
   ```
-  Название бандла определяется по названию контроллера: слова разделяются дефисами, приводятся к нижнему регистру, удаляется суффикс "controller" и добавляется префикс `p-` (например, `MainPageController` → `p-main-page`).
-1. Собственный мэппер - есть возможность написать свой класс мэппера и указать его название в параметре `Mapper`:
+  The bundle name is formed by controller name: all words are separated by dashes, all letters changed to lower case, the "controller" suffix is deleted and `p-` prefix is added (for example, `MainPageController` → `p-main-page`).
+1. The custom mapper - you can write your own mapper class and set it's name in `Mapper` parameter:
 
   ```xml
   <bemSettings Mapper="MyApplication.MyNamespace.InnerNamespace.MyMapperClass" />
   ```
-Класс мэппера должен быть унаследован от базового класса [System.Web.Bem.BundleMappers.Mapper](System.Web.Bem/BundleMappers/Mapper.cs) и реализовывать метод `abstract string GetBundleName(ControllerContext context)`, получающий на вход контекст запроса и возвращающий название бандла. Также, при желании, можно переопределить метод `virtual string GetBundlePath(string bundleName)`, возвращающий по названию бандла путь к файлу с bemhtml шаблонами (по умолчанию формируется путь `<RootDir>\<bundleName>\<bundleName>.bemhtml.js`)
+Your mapper class must be inherited from the base class named [System.Web.Bem.BundleMappers.Mapper](System.Web.Bem/BundleMappers/Mapper.cs) and it must implement the `abstract string GetBundleName(ControllerContext context)` method, which receives a request context as an input parameter and returns the bundle name for this request. Also you can redefine the `virtual string GetBundlePath(string bundleName)` method, which returns the bemhtml-bundle full path by the bundle name (by default the path `<RootDir>\<bundleName>\<bundleName>.bemhtml.js` is formed).
 
-Также в настройках можно задать корневую папку для БЭМ-бандлов проекта (по умолчанию, `~/Bem/desktop.bundles`):
+Also you can setup the default root directory of BEM-bundles (`~/Bem/desktop.bundles` by default):
 
 ```
 <bemSettings RootDir="~/public" />
 ```
 
-### Подключение библиотек блоков
+### External block libraries usage
 
-Вы можете подключать в свой проект сторонние библиотеки с блоками и использовать их. Для этого скопируйте файлы блоков в свой проект и добавьте новые папки с блоками в список уровней переопределения.
+You can add to your project third-party block libraries and use them. For that you should copy the blocks files into your project and add the copied folders to the list of redefinition levels.
```

Comment:
You can add third-party block libraries to your project and use them. To do this, copy the block files